packages/main/src/backend/import/importTransactions.ts

```diff
@@ -193,14 +169,12 @@ async function postProcessTransactions(
 
 function enrichTransaction(transaction: Transaction, companyId: string, accountNumber: string): EnrichedTransaction {
   const hash = calculateTransactionHash(transaction, companyId, accountNumber);
-  // const category = categoryCalculation.getCategoryNameByTransactionDescription(transaction.description);
-  const enrichedTransaction: EnrichedTransaction = {
+  return {
     ...transaction,
     accountNumber,
     // category,
```

Comment:
Oh I see we don't have a category calculation anymore.

Anyway, please return the comment back.

Yes, I agree that developer should avoid code comments, but this project is in low maintainance and I prefer to keep such comments for the next time I will search something.